packages/synapse-react-client/src/components/EntityHeaderTable/EntityHeaderTable.tsx

```diff
@@ -41,17 +37,34 @@ import { EntityHeaderTypeCell } from './EntityHeaderTableCellRenderers'
 import { CheckBoxHeader } from './EntityHeaderTableCellRenderers'
 import { CheckBoxCell } from './EntityHeaderTableCellRenderers'
 import AddAd from '../../assets/icons/AddAd'
-import { EntityFinderModal } from '../EntityFinder/EntityFinderModal'
+import {
+  EntityFinderModal,
+  EntityFinderModalProps,
+} from '../EntityFinder/EntityFinderModal'
 import { VersionSelectionType } from '../EntityFinder/VersionSelectionType'
 import { FinderScope } from '../EntityFinder/tree/EntityTree'
 import { useEntityHeaderTableState } from './useEntityHeaderTableState'
+import { noop, upperFirst } from 'lodash-es'
+import pluralize from 'pluralize'
+
+const DEFAULT_FINDER_CONFIG: EntityFinderModalProps['configuration'] = {
+  selectMultiple: true,
+  versionSelection: VersionSelectionType.DISALLOWED,
+  initialScope: FinderScope.ALL_PROJECTS,
+  initialContainer: 'root',
+}
 
 export type EntityHeaderTableProps = {
   references: ReferenceList
   isEditable: boolean
   onUpdate?: (updatedRefs: ReferenceList) => void // when the references are updated, EntityHeaderTable will call this function with the updated list
   removeSelectedRowsButtonText?: string
   onUpdateEntityIDsTextbox?: (value: string) => void // when the entity IDs text box is updated, this is called
+  /* The word used to describe the items in the table. Default 'entity' */
+  objectNameCopy?: string
+  // If true, the text field where IDs are pasted is hidden, and confirming the entity finder will immediately call `onUpdate`
+  hideTextFieldToPasteValue?: boolean
+  entityFinderConfiguration?: EntityFinderModalProps['configuration']
```

Comment:
In the scope editor, I only want to allow picking containers, so I need to override the finder config